app/controllers/application_controller.rb

```diff
@@ -1851,10 +1856,30 @@ def identify_tl_or_perf_record
     identify_record(params[:id], controller_to_model)
   end
 
-  def assert_privileges(*features)
-    if features.none? { |feature| role_allows?(:feature => feature) }
-      raise MiqException::RbacPrivilegeException, _('The user is not authorized for this task or item.')
```

Comment:
Previously, anything that called `assert_privileges` and didn't log the success/failure would never log anything.